### PR TITLE
[ng] example src reog of attribute-directives, component-styles

### PR DESCRIPTION
**Pull request by chalin**
Here is the first PR contributing to #713

Staged at: 
- https://angulardart-org-dev.firebaseapp.com/angular/guide/attribute-directives
- https://angulardart-org-dev.firebaseapp.com/angular/guide/component-styles

(With tweaks to toh-5, -6, though there should be no visible changes to these two pages.)

### CODE REVIEW

**Comment by kwalrath** [comment]
We should have a review by someone in the know such as @kevmoo or @jakemac53.

File: examples/ng/doc/component-styles/lib/src/hero_app_main_component.dart [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import 'package:angular2/angular2.dart';
 
-import 'hero.dart';
-import 'hero_details_component.dart';
-import 'hero_controls_component.dart';
-import 'quest_summary_component.dart';
+import 'package:component_styles/src/hero.dart';
```
Comment (kevmoo) [inline examples/ng/doc/component-styles/lib/src/hero_app_main_component.dart]: Is this just a rename from IntelliJ? Shouldn't be package: imports...

File: examples/ng/doc/component-styles/lib/src/hero_controls_component.dart [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import 'package:angular2/angular2.dart';
-import 'hero.dart';
+import 'package:component_styles/src/hero.dart';
```
Comment (kevmoo) [inline examples/ng/doc/component-styles/lib/src/hero_controls_component.dart]: Ditto

File: examples/ng/doc/component-styles/lib/src/hero_app_main_component.dart (hunk 1 above)
Comment (chalin) [inline examples/ng/doc/component-styles/lib/src/hero_app_main_component.dart]: Yes. Thanks for catching that.

Comment (chalin) [inline examples/ng/doc/component-styles/lib/src/hero_app_main_component.dart]: Fixed

File: examples/ng/doc/component-styles/lib/src/hero_controls_component.dart (hunk 2 above)
Comment (chalin) [inline examples/ng/doc/component-styles/lib/src/hero_controls_component.dart]: Fixed

**Comment by chalin** [comment]
Done:  dropped the package prefix from `import 'package:component_styles/src/...` occurring in source files under `lib/src`. @kevmoo - PTAL

**Comment by chalin** [comment]
@kwalrath reported offline that @kevmoo approved the latest changes.

**Pull request merged**